app/assets/javascripts/analytics/_virtualPageViews.js

```diff
@@ -1,19 +1,53 @@
 (function (GOVUK) {
   "use strict";
 
-  var sendVirtualPageView = function() {
-    var $element = $(this);
-    var url = $element.data('url');
-    if (GOVUK.analytics  && url){
-      var urlList = url.split("?")
-      urlList[0] = urlList[0] + "/vpv"
-      url = urlList.join("?")
-      GOVUK.analytics.trackPageview(url);
+  var VirtualPageViews = function(){};
+
+  VirtualPageViews.prototype.init = function() {
+
+    var _self = this;
+
+    $('[data-analytics=trackPageView]').each(function(){
+      var $element = $(this);
+      var url = $element.data('url');
+      _self.sendVirtualPageView( url );
+    }).bind(this);
+
+    $('#js-dm-live-search-form .dm-filters').on('change', 'input[type=checkbox]', this.filterSelected);
+
+  };
+
+  VirtualPageViews.prototype.sendVirtualPageView = function( url, name ) {
+
+    if ( GOVUK.analytics  && url ) {
+      var urlList = url.split("?");
+      urlList[0]  = urlList[0] + "/vpv";
+      url         = urlList.join("?");
+      GOVUK.analytics.trackPageview(url, name);
     }
+
   };
 
-  GOVUK.GDM.analytics.virtualPageViews = function() {
-    $('[data-analytics=trackPageView]').each(sendVirtualPageView);
+  VirtualPageViews.prototype.trackFilterAnalytics = function( group, key, value ) {
+
+    var filter_path = '/' + group + '/' + key + '/' + value;
+    var url = '/g-cloud/filters' + filter_path;
+    this.sendVirtualPageView( url, 'Filter' + filter_path.replace(/[/]/g, " - ") );
```

Comment:
I suppose we could find a way to pass in the actual human-readable value, for the page title, but we can come back to that if it turned out to be helpful :)